main.py

```diff
@@ -0,0 +1,49 @@
+from flask import Flask, render_template, request, redirect
+from flask_mysqldb import MySQL
+ 
+app = Flask(__name__)
+
+app.config['MYSQL_HOST'] = 'localhost'
+app.config['MYSQL_USER'] = 'root'
+app.config['MYSQL_PASSWORD'] = 'Shikhar@970'
+app.config['MYSQL_DB'] = 'mysqldb'
+
+mysql = MySQL(app)
+
+@app.route("/")
+def signup_page():
+    return render_template("signup.html")
+
+
+@app.route("/signup", methods=["POST"])
+def manager_page():
+
+    role = request.form["role"]
+    if role == "manager":
+            if request.method == "POST":
+                user_data = request.form
+                Name = user_data['name']
+                email = user_data['mail']
+                Password = user_data['password']
+                Role = user_data['role']
```

Comment:
Some variable names are starting with capital letters. Can you make all of them to small letters?

Reply:
Read this when you get free time https://github.com/google/styleguide/blob/gh-pages/pyguide.md

Reply:
Okay subbu